api/catalog/api/utils/check_dead_links/provider_status_mappings.py

```diff
@@ -0,0 +1,23 @@
+"""Per-provider status mappings."""
```

Comment:
Little suggestion: the description could be a bit more specific.
```suggestion
"""Per-provider HTTP status mappings for link availability."""
```